src/custom/components/Header/HeaderMod.tsx

```diff
@@ -11,13 +11,13 @@ import Logo from 'assets/svg/logo.svg'
 import LogoDark from 'assets/svg/logo_white.svg'
 import { useActiveWeb3React } from 'hooks'
 import { useDarkModeManager } from 'state/user/hooks'
-import { useETHBalances } from 'state/wallet/hooks'
+import { useETHBalances /*, useAggregateUniBalance*/ } from 'state/wallet/hooks'
 // import { CardNoise } from 'components/earn/styled'
 // import { CountUp } from 'use-count-up'
-import { ExternalLink } from 'theme'
+import { /*TYPE,*/ ExternalLink } from 'theme'
```

Comment:
Any reason to keep the comment?

Reply:
Not a big one. I just wanted to keep the original code as much as possible. That was in it. Anyways, I'd be ok to remove it or leave it. Let me know what u prefer

Reply:
I don't mind